datalad_registry/templates/overview.html

```diff
@@ -178,35 +191,89 @@ <h2>Search query syntax</h2>
     <div class="pager">
       {{ render_pagination_widget(pagination, '.overview') }}
     </div>
+
+    <!-- Stats Trigger Button -->
+    <button id="ShowStats">Show Stats</button>
```

Comment:
so they aren't shown by default -- is that because it needs notable backend/DB effort to get them or for another reason?

Reply:
could you may be show screenshots of what it would look like?

Reply:
> so they aren't shown by default -- is that because it needs notable backend/DB effort to get them or for another reason?

That behavior is result of a solely stylistic choice, which maybe be a very good choice in the hindsight.  At https://github.com/datalad/datalad-registry/pull/353/commits/69490edd64020c7c5be948e442906875e9550b3f, the stats are printed directly at the end of the overview page.

Reply:
> could you may be show screenshots of what it would look like?

The changes had been applied to the Typhon instance yesterday, so you can try out the stats feature by accessing the web service on the Typhon instance.  Here are the screenshots though.


![Screenshot 2024-04-20 at 1 01 01 PM](https://github.com/datalad/datalad-registry/assets/12135617/d9c91d85-be83-4cc0-b244-443c7ae0d637)
![Screenshot 2024-04-20 at 1 01 24 PM](https://github.com/datalad/datalad-registry/assets/12135617/65b03023-d0b3-4836-9088-23b24298a14e)

Reply:
Quick one: please use humanize for the displayed information (in API just size).
I think with showing which summary sentence with total size and number of files, and then adding button "show details" with this modal

Reply:
Also for me modal doesn't render correctly at 100%:

![image](https://github.com/datalad/datalad-registry/assets/39889/12677974-6351-4368-9f89-ca88d93c4f7a)

Reply:
Fixed with https://github.com/datalad/datalad-registry/pull/353/commits/000a30d1f13fba67b5326ace1de9ec425f022055.